frontend/src/widgets/RegionalGoalDashboard/GoalStatusChart.js

```diff
@@ -0,0 +1,4 @@
+import withWidgetData from '../withWidgetData';
+import { GoalStatusChart } from '../GoalStatusGraph';
+
```

Comment:
Should we be wrapping withWidget data around a reusable component.

This just looks weird to me.

Reply:
yeah I did this. The frontend component that shows the data is the same, but this needs to retrieve data from a different endpoint. This lets us reuse the existing frontend while feeding it a new backend

Reply:
👍 Maybe a comment then.

Reply:
```js
// eh